test/dynamic_shape_model_test.py

```diff
@@ -0,0 +1,79 @@
+from sklearn.datasets import make_blobs
+import torch, torch_xla
+import torch_xla.core.xla_model as xm
+import numpy
+
+pd = torch._C._EnablePythonDispatcher()
```

Comment:
I wonder what this line does

Reply:
it enables us to run python implementations of `CompositeAutogradImplicit` ops.
`CompositeAutogradImplicit` means we don't have an explicit backward formula for an op instead an op is composed of a bunch of ops that do have backward formulas and combines this formulas is equivalent to differentiating the op explicitly.